### PR TITLE
Precompute image aspect ratios

### LINKED ISSUE #3: Sample project scrolling is choppy
Have you test it ?
On a Samsung Galaxy s6 with L android it does not work at all
scroll is very choppy images are disappeared as you scroll back again.
I can provide a screen capture if you wish

### PR DESCRIPTION
**Pull request by JVillella**
This PR precomputes the image aspect ratios required for layout in the sample project. @a64adam, @AngleV please have a look.

Fixes #3

### CODE REVIEW

**Comment by a64adam** [comment]
Looks good to me!

**Comment by AngleV** [comment]
@JVillella 
It is much better now but i think it still needs some improvement.Scrolling slowly it works nice but if you try to fling you can see that there is a stuttering

http://sendvid.com/59v27puj

**Comment by JVillella** [comment]
@AngleV the scrolling looks smooth in the video, but the caching doesn't seem to be so great. I'll increase the mem cache, and lower the resolution of the photos in the sample project so it runs a little quicker. I want to make the note that this has nothing to do with the layout behaviour, it's the image caching in the sample app.

**Comment by JVillella** [comment]
@AngleV Please let me know if this works better for you, I will merge it in afterwards.

File: greedo-layout-sample/src/main/java/com/fivehundredpx/greedo_layout_sample/App.java [hunk 1]
```diff
@@ -1,25 +1,38 @@
...
-        // Increase picasso cache size
-        OkHttpDownloader okHttpDownloader = new OkHttpDownloader(this, PICASSO_DISK_CACHE_SIZE);
         Picasso picasso = new Picasso.Builder(this)
-                .downloader(okHttpDownloader)
+                .memoryCache(new LruCache(calculateMemoryCacheSize()))
                 .build();
         Picasso.setSingletonInstance(picasso);
     }
+
+    int calculateMemoryCacheSize() {
```
Comment (beltex) [inline greedo-layout-sample/src/main/java/com/fivehundredpx/greedo_layout_sample/App.java]: Minor: `private` ?

**Comment by beltex** [comment]
Very smooth! :+1:

**Pull request merged**